edelphi/src/main/webapp/_scripts/gui/panel/queryblockcontroller.js

```diff
@@ -2792,6 +2794,8 @@ QueryCommentsController = Class.create({
 
     Event.observe(saveButtonElement, "click", function(event) {
       Event.stop(event);
+      saveButtonElement.writeAttribute("disabled", "disabled");
+      
       var element = Event.element(event);
       var newText = textEditorElement.value;
       startLoadingOperation('query.comment.updatingComment');
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') "startLoadingOperation" does not exist. Change its name or declare it so that its usage doesn't result in a "ReferenceError". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=javascript%3AS3827)